linkml_runtime/dumpers/rdf_dumper.py

```diff
@@ -81,7 +81,77 @@ def dumps(self, element: YAMLRoot, contexts: CONTEXTS_PARAM_TYPE = None, fmt: Op
         Convert element into an RDF graph guided by the context(s) in contexts
         :param element: element to represent in RDF
         :param contexts: JSON-LD context(s) in the form of a file or URL, a json string or a json obj
-        :param fmt: rdf format
+        :param fmt: RDF format
         :return: rdflib Graph containing element
         """
         return self.as_rdf_graph(element, contexts).serialize(format=fmt).decode()
+
+@click.group()
+def cli():
+    pass
+
+@cli.command('as_rdf_graph')
+@click.option('-e', '--element', help='LinkML object to be emitted.')
+@click.option('-c', '--contexts', help='JSON-LD context(s)')
```

Comment:
having `(s)` indicates this can be multivalued but this seems a single valued option?

Reply:
I missed it, sorry about that. Does `@click.argument('contexts', required=True')` make sense?